modules/library/render/src/test/resources/org/geotools/renderer/lite/test-data/strikethroughStyle.sld

```diff
@@ -0,0 +1,146 @@
+                            <CssParameter name="fill">#00FF00</CssParameter>
+                        </Fill>
+                        <VendorOption name="strikethroughText">true</VendorOption>
+                    </TextSymbolizer>
+                </Rule>
+                <Rule>
+                    <ogc:Filter>
+                        <ogc:PropertyIsEqualTo>
+                            <ogc:Function name="in2">
+                                <ogc:Function name="id"/>
+                                <ogc:Literal>Line.5</ogc:Literal>
+                                <ogc:Literal>Line.8</ogc:Literal>
+                            </ogc:Function>
+                            <ogc:Literal>true</ogc:Literal>
+                        </ogc:PropertyIsEqualTo>
+                    </ogc:Filter>
+                    <TextSymbolizer>
+                        <Label>
+                            <ogc:PropertyName>name</ogc:PropertyName>
+                        </Label>
+                        <Font>
+                            <CssParameter name="font-family">Bitstream Vera Sans</CssParameter>
+                            <CssParameter name="font-style">italic</CssParameter>
+                            <CssParameter name="font-size">25</CssParameter>
+                        </Font>
+                        <LabelPlacement>
+                            <LinePlacement/>
+                        </LabelPlacement>
+                        <Fill>
+                            <CssParameter name="fill">#00FFFF</CssParameter>
+                        </Fill>
+                        <VendorOption name="followLine">true</VendorOption>
+                        <VendorOption name="strikethroughText">true</VendorOption>
+                    </TextSymbolizer>
+                </Rule>
+                <Rule>
+                    <ogc:Filter>
+                        <ogc:PropertyIsEqualTo>
+                            <ogc:Function name="id"/>
+                            <ogc:Literal>Line.9</ogc:Literal>
+                        </ogc:PropertyIsEqualTo>
+                    </ogc:Filter>
+                    <TextSymbolizer>
+                        <Label>
+                            <ogc:PropertyName>name</ogc:PropertyName>
+                        </Label>
+                        <Font>
+                            <CssParameter name="font-family">Bitstream Vera Sans</CssParameter>
+                            <CssParameter name="font-size">30</CssParameter>
+                        </Font>
+                        <LabelPlacement>
+                            <LinePlacement/>
+                        </LabelPlacement>
+                        <VendorOption name="autoWrap">3</VendorOption>
+                        <VendorOption name="strikethroughText">true</VendorOption>
+                    </TextSymbolizer>
+                </Rule>
+            </FeatureTypeStyle>
+        </UserStyle>
+    </NamedLayer>
+</StyledLayerDescriptor>
```

Comment:
Ah, ok: https://stackoverflow.com/questions/13478373/how-to-remove-empty-line-from-the-end-of-new-class-eclipse